launchlib/config_test.go

```diff
@@ -41,9 +44,13 @@ args:
 		ConfigVersion: 1,
 		MainClass:     "mainClass",
 		JavaHome:      "javaHome",
-		Classpath:     []string{"classpath1", "classpath2"},
-		JvmOpts:       []string{"jvmOpt1", "jvmOpt2"},
-		Args:          []string{"arg1", "arg2"}}
+		Env: map[string]string{
+			"SOME_ENV_VAR":  "/etc/profile",
+			"OTHER_ENV_VAR": "/etc/redhat-release",
```

Comment:
should test parsing behaviour for non-supported placeholder. in particular does `"%%.FOO%%BAR"` resolve to `""` or to `"%%.FOO%%BAR"`, or does it yield an exception? i'd favour the latter.